android/src/main/java/io/github/droidkaigi/feeder/notification/KaigiMessagingService.kt

```diff
@@ -0,0 +1,22 @@
+package io.github.droidkaigi.feeder.notification
+
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class KaigiMessagingService : FirebaseMessagingService() {
+
+    override fun onCreate() {
+        NotificationUtil.createNotificationChannel(this)
+    }
+
+    override fun onNewToken(token: String) {
+        // TODO: 2021/02/28 implement sending server
```

Comment:
Will you implement this? If not, can you create an issue for this? 🙏

Reply:
yes! ok, i will diassemble and make pr